versions/next/asyncapi.md

```diff
@@ -672,6 +672,7 @@ Field Name | Type | Description
 ---|:---:|---
 <a name="parameterObjectName"></a>name | `string` | The name of the parameter.
 <a name="parameterObjectDescription"></a>description | `string` | A verbose explanation of the parameter. [CommonMark syntax](http://spec.commonmark.org/) can be used for rich text representation.
+<a name="parameterObjectMultiLevel"></a>multiLevel | `boolean` | Whether a parameter spans across multiple levels or not. Defaults to `false`.
```

Comment:
This appears to be the only relevant use of the word level(s) in the spec. I.e. it is undefined. Can we be clearer as to a) what kind of parameters can be defined as `multiLevel` - i.e. is it only parameters that act as wildcards and b) what a 'level' is in this context?

Reply:
You're right, it should be clearer. To answer your points:

A) Yes, only parameters can act as wildcards, and this property is just for wildcard parameters.

B) Check out this article for context: https://www.hivemq.com/blog/mqtt-essentials-part-5-mqtt-topics-best-practices/. We should come up with a good explanation here.

Thanks!

Reply:
Actually, regarding A, all parameters are wildcards.

Reply:
@MikeRalphson can you have a look at the PR fixing this? https://github.com/asyncapi/asyncapi/pull/187. I'm not happy with my explanation, to be honest 😅